creator-node/src/middlewares.js

```diff
@@ -54,7 +54,7 @@ async function syncLockMiddleware (req, res, next) {
 
 /** Blocks writes if node is not the primary for audiusUser associated with wallet. */
 async function ensurePrimaryMiddleware (req, res, next) {
-  if (config.get('isUserMetadataNode')) {
+  if (config.get('isUserMetadataNode') || req.body.forceWrite) {
```

Comment:
can we add a comment here saying if node is UM or assigning CN for first time we ignore this middleware?

Reply:
down to rename this as explicitly verbose as possible--if/once we agree on whether this is a good idea or not